net.audiorelay.AudioRelay.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.audiorelay.AudioRelay</id>
+  
+  <name>AudioRelay</name>
+  <summary>Stream audio between your devices</summary>
+  <developer_name>Asapha Halifa</developer_name>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  
+  <description>
+    <p>
+      Turn your phone into a microphone or speakers for PC. Use Wi-Fi or USB to share audio to multiple devices with low delay.
+    </p>
+  </description>
+    
+  <launchable type="desktop-id">net.audiorelay.AudioRelay.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://imgur.com/9o0GqbK.png</image>
```

Comment:
imgur is not a reliable location for screenshots.

Reply:
I have updated the link to the app's official docs.